### DISCUSSION TITLE
logic - waves - slate

### DISCUSSION THREAD

**Discussion by Erikandersglass** [apohl79/audiogridder#1277] [Help]
trying to put this in the best place but running gridder in logic, 
- some waves plugins show as flipped and jumbled UI. 
-it is also impossible to use existing presets from the plugins.
- when using dual mono instances there is no way to couple or toggle L R it just shows one screen without controls. 
- slate digital have been failing to scan.
- stock logic plugs can be used but the old stock are there.

otherwise gridder has saved my life! thank you for making it!!!!!!

**Comment by apohl79** [comment]
Cheers for reporting this. The waves issue is known and not solved yet.

Can you elaborate a little more on the dual mono issue?

**Comment by Erikandersglass** [comment]
Thanks  for the reply ! When selecting use of dual mono in logic there is
usually a tab at the top of the plugin that allows switching from L and R
controlls. This does not show up in gridder at all so dual mono instances
can't be used.
